packages/drafts-realm/.realm.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Drafts Workspace",
+  "backgroundURL": "https://i.postimg.cc/VNvHH93M/pawel-czerwinski-Ly-ZLa-A5jti-Y-unsplash.jpg",
+  "iconURL": "https://i.postimg.cc/L8yXRvws/icon.png"
```

Comment:
I'm not using IPFS since i'm not aware of a free pinning service that we can use